com.dec05eba.git.gpu_screen_recorder.yml

```diff
@@ -0,0 +1,79 @@
+app-id: com.dec05eba.git.gpu_screen_recorder
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: gpu-screen-recorder-gtk
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --filesystem=home
```

Comment:
why does it need access to the whole home directory?

Reply:
Im not sure how to solve this. The application needs access to ~/.config/gpu-screen-recorder/config but also needs to be able to write to the path chosen by the user in the file/directory picker

Reply:
The Gtk filepicker gives the App permission to the chosen file, so you don't need full home access.

Reply:
That doesn't seem to work in this case because it's two separate programs. The gui is one program (gpu-screen-recorder-gtk) but the program that needs permission to create the video file is a cli application (gpu-screen-recorder) that the gui launches (the cli is launched as a child process). The file doesn't get created without --filesystem=home.
Is this supposed to work if the program is a child process or do I need to solve this some other way? I read the sandbox documentation but I didn't see any solution to this.